### PR TITLE
Filter option mobile view

### PR DESCRIPTION
**Pull request by peeyush-webkul**
## Issue Reference
#7397

### CODE REVIEW

**Comment by Amitk-webkul** [comment]
@peeyush-webkul 
Now It is working fine. [Here](https://prnt.sc/Eo8bL8qrDHK5)

**Comment by Sanjaybhattwebkul** [comment]
@peeyush-webkul 
Resolve the **duplicate commits.**

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Velocity/src/Resources/assets/js/UI/components/header-mobile.vue [hunk 1]
```diff
@@ -372,12 +376,33 @@ export default {
...
         },
+
+        screenSize: function () {
+            if(this.screenSize >= 600 && this.screenSize < 992) {
+                this.isTabletMobileScreenSearch=true;
+            } else {
+            this.isTabletMobileScreenSearch=false;
+            }
+
+        }
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/js/UI/components/header-mobile.vue:387]: Improve this.

File: packages/Webkul/Velocity/src/Resources/assets/js/UI/components/header-mobile.vue [hunk 2]
```diff
@@ -372,12 +376,33 @@ export default {
...
+        window.addEventListener('resize', this.handleResize);
+
+        this.handleResize();
+
         this.getMiniCartDetails();
 
         this.updateHeaderItemsCount();
+
+
+    },
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/js/UI/components/header-mobile.vue:402]: Remove all extra lines and aligned them properly.

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 3]
```diff
@@ -215,6 +219,24 @@
...
+                    }
+
+                    .col-7 {
+                        > div {
+                            margin-left: 8%;
+                            margin-right: -22%;
+                        }
+                    }
+                }
+
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:239]: Remove extra line.

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 4]
```diff
@@ -625,6 +647,90 @@
...
                     }
+
+                    >div {
+                        &:nth-child(2) {
+                            z-index: 2;
+                        }
+                    }
+
+                    @media only screen and (max-width: 700px) {
+
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:658]: Same here.

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 5]
```diff
@@ -625,6 +647,90 @@
...
+                                margin-right: -7px;
+                            }
+                        }
+
+                        #limit-by {
+                            >select {
+                                padding-left: 4px !important;
+                            }
+                        }
+
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:708]: Same here.

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 6]
```diff
@@ -625,6 +647,90 @@
...
+                        #limit-by {
+                            >select {
+                                padding-left: 4px !important;
+                            }
+                        }
+                    }
+
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:733]: Same here.

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 7]
```diff
@@ -1071,3 +1170,12 @@
         }
     }
 }
+
+@media only screen and (max-width: 3092px) {
+    .selective-div {
+        width: -webkit-fill-available !important;
+        -webkit-appearance: none !important;
+
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:1178]: Same here.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by devansh-webkul** [comment]
I am seeing old commits also. Please check.

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss [hunk 8]
```diff
@@ -625,6 +646,85 @@
...
+                                margin-left: 14px;
+                            }
+                            &:nth-child(3) {
+                                margin-left: 13px;
+                                margin-right: -13px;
+                            }
+                            &:nth-child(4) {
+                                margin-left: 4px;
+                                margin-right: -4px;
+                            }
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/media.scss:719]: Add a new line after each closing bracket.

**Comment by Amitk-webkul** [comment]
@peeyush-webkul 
on the filter page, the page width is not set. [Here](https://prnt.sc/CalcJALkP8QT)

**Comment by Amitk-webkul** [comment]
@peeyush-webkul 
Not fixed for the sub-category. [Here](https://prnt.sc/_VmTmd9uyoz6)

**Comment by Amitk-webkul** [comment]
@peeyush-webkul 
It is working fine.[Here](https://prnt.sc/f7mCDT5utu8l)

**Review by Amitk-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**